ROX-27719: Enable FF for RHCOS node scanning with scanner V4 (#14154)
stackrox/stackrox: CHANGELOG.md

### Added Features

- ROX-26847: RHCOS Node Scanning with Scanner V4
    - ROX-27719: is now enabled by default on all secured clusters and will be preferred over the Stackrox Scanner if Scanner V4 is installed and connected to Central.

>     - ROX-25625: can now detect vulnerabilities for the containerized image of the RHCOS itself.

Reviewer: ```suggestion
    - ROX-25625: can now detect vulnerabilities for its own containerized RHCOS image.
```
[nit] better wording

Author: It is not it's own image 😅 RHCOS runs on the nodes and Scanner V4 runs in an image. Both are RHEL9-based, but I don't think that is enough to say that Scanner V4's own image is RHCOS.

Author: But I get the confusion caused by the word `itself`. What I meant by this is that there is no rpm package called `rhcos`, so the previous scanner V4 (in 4.6) could not report vulns on it. Now we add that, so `rhcos` is represented by a package named the same as the operating system (this is intended, as VEX publishes data under this name). 
Having said that, would you like to propose another rephrasing maybe?

Author: @jschnath I would like to merge Today. Pinging you in case you want to propose another rephrasing. 
If not, I would merge it as it is - it does not need to be ideal, as release notes are not copy-pasted 1:1, but edited by the doc writers before publishing.
